### ISSUE TITLE
file variable cannot be passed in Table driven scenario

### ISSUE THREAD

**Issue by z04jmath** [getgauge/gauge#1746]
Using 
```
Gauge version: 1.1.3
Commit Hash: 9d3a3d5

Plugins
-------
html-report (4.0.10)
js (2.3.10)
json-report (0.3.3)
screenshot (0.0.1)
xml-report (0.2.3)
```

When using table driven scenario like this

```md
## temp
| case | data |
| - | - |
| 1 | <file:Data/json/1.json> |
| 2 | <file:Data/json/2.json> |
* test <data>
```

Content of file is not putted into data variable, it contains `file:Data/json/1.json` instead of its content

**Comment by sriv** [comment]
Yes, currently table driven scenarios is an experimental feature, and has limitations. This is one of those limitations and needs to be addressed.

**Comment by BugDiver** [comment]
Fixed via #1765
